cuda/io/src/debayering.cu

```diff
@@ -114,8 +114,8 @@ namespace pcl
     OpenNIRGB DebayerBilinear<Storage>::operator () (int index) const
     {
     	// get position
-    	int xIdx = index % width;
-    	int yIdx = index / width;
+    	unsigned xIdx = index % width;
```

Comment:
unsigned int?

Reply:
Yes, exactly `unsigned int`. Should I change this to `unsigned int` to make the declaration more explicit?

Reply:
I apologize for being used to declare by a shorthand, which may be not nice for our project.